report-api/src/main/resources/template/affiliation.sparql

```diff
@@ -16,8 +17,12 @@ WHERE {
   GRAPH ?g {
     ?uri a :Publication ;
          :identifier ?publicationIdentifier ;
+         :status ?publicationStatus ;
          :modifiedDate ?modifiedDate ;
          :entityDescription/:contributor ?contributor .
+
+    BIND(REPLACE(STR(?publicationStatus), "^.*/([^/]*)$", "$1") AS ?status)
```

Comment:
We want a string here, it should be a URI — which I am not sure if it is in the data at the moment